### DISCUSSION TITLE
How to export a Photo library to a Linux environment?

### DISCUSSION THREAD

**Discussion by jairmedeiros** [RhetTbull/osxphotos#1521] [Q&A] [answered]
Hey guys! I didn't get how to run this tool in a Linux environment (sorry for that). But, what are the pre-requirements to run the osxphotos? I don't have anything Apple-related on my Linux environment (like iTunes, Apple Photos, etc).

Is it possible to get the photos directly from an API on the web? Like a Google Takeout?

Unfortunately, I don't have a macOS machine to create a VM, for instance. So, I'm finding a way to get all my media from Apple Photos.

**Comment by RhetTbull** [comment, marked as answer]
osxphotos can be used on linux but only to access an Apple Photos Library (for example, one mounted via a Mac or an external drive). For your use case, there's a different app (by a different author) called [iCloud Photos Downloader](https://github.com/icloud-photos-downloader/icloud_photos_downloader) that will download photos directly from iCloud.

**Reply by jairmedeiros** [reply]
Got it! Thanks so much!!
